router/internal/expr/expr.go

```diff
@@ -93,6 +96,18 @@ type RequestAuth struct {
 	Scopes          []string       `expr:"scopes"`
 }
 
+// Subgraph Related
+type Subgraph struct {
```

Comment:
Following the spec (except for the "Trace" attribute) from https://github.com/wundergraph/cosmo/pull/1481